### PR TITLE
return nan for negative input, plus test (fix #21)

### LINKED ISSUE #21: Negative input should throw np.nan
Inputs like v_m, rho_m, mu_m, Q_s..
Maybe the simplest thing would be to say
`if E < 0: return np.nan` for all erosion models? This way, any negative input will throw np.nan...

### PR DESCRIPTION
**Pull request by PoyiOstona**

### CODE REVIEW

File: tests/test_erosion.py [hunk 1]
```diff
@@ -50,8 +45,8 @@ def test_validate_inputs(caplog):
                  in s.message for s in info)
 
     # Test Q_s boundaries
-    kwargs = {'Q_s': 1, 'v_m': 1, 'D': .1}
-    for illegal_input in [-1, 555]:
+    kwargs = {'Q_s': 1, 'rho_p': 1, 'v_m': 1, 'D': .1}
```
Comment (knudsvik) [inline tests/test_erosion.py]: Why have you inserted `rho_p`? It is added in line 33/34

File: pysand/erosion.py [hunk 2]
```diff
@@ -37,9 +37,11 @@ def validate_inputs(**kwargs):
         if i in kwargs:
             if not isinstance(kwargs[i], (float, int, np.ndarray, pd.Series)) or np.isnan(kwargs[i]):
                 raise exc.FunctionInputFail('{} is not a number or pandas series'.format(i))
+            if not kwargs[i] >= 0:
+                return True
```
Comment (knudsvik) [inline pysand/erosion.py]: we should throw a warning here as well, don't you think? If any erosion results throw np.nan it's not possible to see why..

File: tests/test_erosion.py (hunk 1 above)
Comment (PoyiOstona) [reply tests/test_erosion.py]: Did not notice. Will remove it then :)

File: pysand/erosion.py (hunk 2 above)
Comment (PoyiOstona) [reply pysand/erosion.py]: Yes, throwing a warning message on this is a good idea.
What do you mean by not possible to see? For negative input the result is nan

**Pull request merged**